frontend/src/views/Transactions/TransferDetailsView.jsx

```diff
@@ -0,0 +1,193 @@
+import { useNavigate } from 'react-router-dom'
+import {
+  Typography,
+  Stepper,
+  Step,
+  StepLabel,
+  Stack,
+  List,
+  ListItem
+} from '@mui/material'
+import BCBox from '@/components/BCBox'
+import BCButton from '@/components/BCButton'
+import { decimalFormatter } from '@/utils/formatters'
+import { BTN_RESCIND_TRANSFER, BTN_APP_CANCEL } from '@/constants/langEnUs'
+// Icons
+import arrowRight from '@/assets/icons/arrowRight.svg'
+import arrowLeft from '@/assets/icons/arrowLeft.svg'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+  faCircle,
+  faArrowLeft,
+  faTrash
+} from '@fortawesome/free-solid-svg-icons'
+// Sub components
+import { OrganizationBadge } from './components/OrganizationBadge'
+import { demoData } from './components/demo'
+import { AttachmentList } from './components/AttachmentList'
+import { Comments } from './components/Comments'
+
+const idirSteps = ['Draft', 'Sent', 'Submitted', 'Recorded']
+
+export const TransferDetailsView = () => {
+  const navigate = useNavigate()
```

Comment:
please add some commented out code to get the transfer details from the api. Use the orgView as an example

```js
export const ViewOrg = () => {
  const [showActive, setShowActive] = useState(true)
  const navigate = useNavigate()
  const { orgID } = useParams()
  const { data: currentUser, isLoading: isCurrentUserLoading } =
    useCurrentUser()
  const { data: orgData, isLoading } = useOrganization(orgID)
  const handleEditClick = () => {
    navigate(ROUTES.ORGANIZATIONS_EDIT.replace(':orgID', orgID), {
      state: { orgID, isEditMode: true }
    })
  }
```
please also add a useTransfer hook that prepares us to be able to get transfer data from the api

```js
import { useApiService } from '@/services/useApiService'
import { useQuery } from '@tanstack/react-query'
import { useCurrentUser } from './useCurrentUser'

export const useOrganization = (orgID, options) => {
  const client = useApiService()
  const { data: currentUser } = useCurrentUser()
  const id = orgID ?? currentUser.organization.organization_id

  return useQuery({
    queryKey: ['organization', id],
    queryFn: async () => (await client.get(`/organizations/${id}`)).data,
    ...options
  })
}

Reply:
updated